schedule/yast/sle/flows/default_s390x_kvm.yaml

```diff
@@ -0,0 +1,61 @@
+---
+# Default ordered sequence of steps to be optionally overwritten for s390x
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+access_beta:
+  - installation/access_beta_distribution
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
+  - installation/licensing/accept_license
+registration:
+  - installation/registration/skip_registration
+extension_module_selection:
+  - installation/module_registration/skip_module_registration
+add_on_product:
+  - installation/add_on_product_installation/accept_add_on_installation
+system_role:
+  - installation/installation_settings/validate_ssh_service_enabled
+  - installation/installation_settings/open_ssh_port
+guided_open: []
+guided_hard_disks: []
+guided_scheme: []
+guided_filesystem: []
+suggested_partitioning:
+  - installation/partitioning/accept_proposed_layout
+clock_and_timezone:
+  - installation/clock_and_timezone/accept_timezone_configuration
+local_user:
+  - installation/authentication/use_same_password_for_root
+  - installation/authentication/default_user_simple_pwd
+software:
+  - installation/select_patterns
```

Comment:
this goes into the schedule, not the default.

Reply:
changed.